lib/money/money.rb

```diff
@@ -70,7 +70,7 @@ def initialize(value = 0, currency = nil)
     raise ArgumentError if value.respond_to?(:nan?) && value.nan?
     @currency = Helpers.value_to_currency(currency)
     @value = Helpers.value_to_decimal(value).round(2)
```

Comment:
was going to do that with #73 but yea this won't work with 3 digit currencies. Will cherry-pick that commit here